bots/tests-scan

```diff
@@ -80,10 +80,23 @@ def main():
                         help='Don''t actually change anything on GitHub')
     parser.add_argument('--repo', default=None,
                         help='Repository to scan and checkout.')
+    parser.add_argument('--module-repo', default=None,
+                        help='Repository to scan and incorporate into Cockpit.')
+    parser.add_argument('--module-updater', default=None,
+                        help='Program to call to update Cockpit with the module.')
```

Comment:
Here I think "updater" makes sense. Nitpick: These two new options that get introduced here, and the `--repo` one have help texts ending with a period, while the other/older ones don't.